package.json

```diff
@@ -25,29 +25,29 @@
     "build": "tsc -d",
     "clean": "rm -rf ./generated ./dist ./build",
     "pretest": "./scripts/regenerate_protos.sh && tsc",
-    "lint": "eslint **/*.ts",
-    "lint:fix": "eslint **/*.ts --fix",
+    "lint": "eslint . --ext .ts --fix",
     "test": "npm run lint && nyc mocha",
     "prepublishOnly": "npm test && tsc -d",
     "webpack": "npm run clean && ./scripts/regenerate_protos.sh && tsc && npx webpack"
   },
   "devDependencies": {
     "@types/chai": "4.2.7",
     "@types/mocha": "5.2.7",
-    "@typescript-eslint/eslint-plugin": "1.13.0",
-    "@typescript-eslint/parser": "1.13.0",
+    "@typescript-eslint/eslint-plugin": "^2.16.0",
```

Comment:
In general, I'm OCD so I prefer version locking. Any reason you're unlocking these versions?

Reply:
I like getting new functionality without having to think about it.

I feel you on version locking for core/runtime dependencies, but I don't feel it's necessary for our linters. I'm happy to version lock if that is a strong preference.

Reply:
Nope that's a fine way to look at it. Carry on.